processor/transformer/transformer.go

```diff
@@ -68,7 +68,9 @@ type MetadataT struct {
 	EventType               string   `json:"eventType"`
 	SourceDefinitionID      string   `json:"sourceDefinitionId"`
 	DestinationDefinitionID string   `json:"destinationDefinitionId"`
-	SourceDefinitionType    string
+	TransformationID        string   `json:"transformationId"`
+	TransformationVersionID string   `json:"transformationVersionId"`
+	SourceDefinitionType    string   `json:"-"`
```

Comment:
any reason why  `json:"-"`

Reply:
Basically it doesn't appear in json after marshalling, figured it wouldn't be needed for transformer